P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+        variety of attributes. Common attributes used in conjunction with <gi>post</gi> include
+          <att>who</att>, <att>synch</att>, <att>type</att>, <att>subtype</att>, <att>rend</att>,
+        and <att>xml:id</att>.</p>
+    </div>
+    <div xml:id="CMCcmcpostatts">
+      <head>Attributes specific to CMC <gi>post</gi></head>
+      <p>Three attributes pertain specifically to <gi>post</gi>:
+        <specList>
+          <specDesc key="post" atts="modality replyTo indentLevel"/>
+        </specList>
+        The type of the content of a post (i.e., whether the content is written, an
+        image, a video clip, etc.) is indicated by the child elements of the <gi>post</gi>. (E.g., a
+        <gi>post</gi> might have a child <gi>p</gi>, or a child <gi>figure</gi> with a
+        <gi>graphic</gi>, or a child <gi>figure</gi> with a <gi>media</gi>, or some combination
+        thereof.) How that content was created — in particular whether it was recorded speech or not
+        — may be described with the <att>modality</att> attribute. Because spoken language differs
+        significantly from written language the suggested values only separate <val>written</val>
+        modality — which covers all cases other than spoken natural language — from
+        <val>spoken</val>. The use of <att>modality</att> is recommended but not required.
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <post modality="written" generatedBy="human" synch="#t005" who="#A06" xml:id="cmc_post09">
+            <figure type="image" generatedBy="human">
+              <desc xml:lang="en">screenshot of the google search for hairdresser "Pasha's Haare'm"
+                with the average google rating (4,5 of 5 stars), the address, the phone number, and
+                the opening hours.</desc>
+            </figure>
+          </post>
+        </egXML>
+      </p>
+      <p>The <att>replyTo</att> attribute is used to capture information drawn from the original
+        metadata associated with a post that asserts to which previous post the current post is a
+        response, or to which previous post it refers. This metadata is included by many, but not
+        all, CMC environments, when the user executes a formal reply action (e.g., by clicking or
+        tapping a reply button). This attribute should not be used to encode interpreted or inferred
+        reply relations based on linguistic cues or discourse markers.</p>
+      <p>The <att>replyTo</att> attribute indicates the replied-to or referred-to posts by providing
+        one or more pointers to them. In the following example, reply references in the source
+        indicate that the first <gi>post</gi> is a reply to an initial post that is not part of the
+        example, the second is a reply to the first, and the third is a reply to the second.
+        <egXML xml:id="e9" xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de" source="#BIB_scilog1">
```

Comment:
Is there an English version of this example, in case the content helps explain the function of the replyTo attribute?